### ISSUE TITLE
Întrebări generale legate de cursul WebDev

### ISSUE THREAD

**Issue by dserban** [dserban/webdev2015fall#2]

**Comment by Canutza** [comment]
Salut,
Ma pot inscrie daca nu mai sunt student ?

**Comment by dserban** [comment]
Salut Mihai,
Da, te poti inscrie.

**Comment by scipianus** [comment]
Neaparat va fi in weekend si, mai mult, in fiecare weekend in ambele zile? Pentru cei din provincie inseamna sa nu mai mearga acasa timp de ceva saptamani si sa aiba toate weekendurile blocate.

**Comment by dserban** [comment]
Salut Ciprian,
Sunt foarte multe constrangeri legate si de sali si de instructori.
Cursul WebDev nu poate avea loc decat in zilele de weekend, din pacate.

**Comment by Saniuta** [comment]
Salut ,
La problema de programare , la fiecare test trebuie sa cautam dupa fiecare cuvant din lista de cuvinte ?

**Comment by neatudarius** [comment]
Salutare,
Intrebarile se pun ca si comentariu la issue pe GitHub.
https://github.com/dserban/webdev2015fall/issues/3

Multumesc pentru intelegere, Darius  


     On Friday, September 11, 2015 8:06 PM, Saniuta <notifications@github.com> wrote:
   

 Salut ,
La problema de programare , la fiecare test trebuie sa cautam dupa fiecare cuvant din lista de cuvinte ?—

**Comment by dserban** [comment]
@Saniuta Da, la fiecare test trebuie luate la rand si cautate toate cuvintele din lista.
@neatudarius A procedat bine din cate vad, adica a postat intrebarea ca issue.

**Comment by menocchio** [comment]
Ar trebui gasita si posibilitatea transmiterii online - chiar daca s-ar mentine probele sau macar inregistrat is pus ulterior online!

**Comment by dumytru** [comment]
@dserban @neatudarius <- Scuze vina mea; am crezut ca i-a trimis mail direct (se vede iconita cum ca Darius a raspuns prin email.)

**Comment by MichelleMorgan** [comment]
Este nevoie de laptop pentru curs/hackathon?

**Comment by dserban** [comment]
@MichelleMorgan Cel mai probabil da. Insa vom face efortul sa gasim o solutie si pentru care nu pot veni cu laptop-ul propriu de acasa, deci sa nu te descurajeze intrebarea de pe formular. Inscrie-te daca intentionezi sa participi la curs.

**Comment by Mihaiii** [comment]
Vor fi cursuri si duminica (ca anul trecut) sau numai sambata? M-am uitat peste orar si sunt acoperite numai zile de sambata - de asta intreb.

**Comment by dserban** [comment]
@Mihaiii
Workshop-urile pot avea loc si sambata si duminica, dupa cum dicteaza circumstantele.
De exemplu, daca intr-un anumit weekend nu gasesc instructor pentru un anumit track, voi tine eu ambele workshop-uri, ceea ce in mod evident inseamna ca vor trebui sa aiba loc in zile diferite.

**Comment by cristiibm** [comment]
Buna ziua,
lipsesc comentarii anterioare?
mai sunt locuri?

**Comment by dserban** [comment]
@cristiibm
La intrebarea "mai sunt locuri" nu pot sa-ti raspund acum, fiindca nu s-au incheiat inscrierile.
Ceea ce pot sa-ti spun este ca locul ti-l creezi singur rezolvand corect problemele date.
Te incurajez sa le rezolvi si sa te inscrii cu solutiile.

**Comment by bogdantalpiga** [comment]
Buna ziua!
In cazul in care inscrierile se pot depune pana la data de 14.10 inclusiv, iar lista cu cei admisi va fi publicata pe 15.10, cand vor avea loc interviurile "face-to-face"?
Multumesc.

**Comment by dserban** [comment]
@bogdantalpiga
Interviurile vor putea avea loc si face-to-face, si prin Google Hangouts, deci nu este o problema foarte mare. Colegii mei din echipa de organizare iti vor da imediat mai multe detalii.

**Comment by neatudarius** [comment]
@bogdantalpiga 
Incepand de maine seara vom trimite email/SMS celor care au rezolvat deja problema. Ii vom chema in zilele de 12,13,14,15. Pana pe data de 15.10.2015 ora 23:59 vom anunta lista finala cu cei admisi.

**Comment by mihaisavezi** [comment]
Salut, cum vom fi anuntati de interviuri ?

**Comment by dserban** [comment]
@mihaisavezi
Nu ai primit e-mail?

**Comment by mihaisavezi** [comment]
@dserban Nu..., poti verifica dupa contul meu github daca ar fi trebuit sa primesc ? Sincer nu mai stiu cu care adresa de mail am completat formularul...

**Comment by dumytru** [comment]
@mihaisavezi Inca nu am trimis mail la cei care au trimis aseara solutia la problema. Inca corectam sursele si vom trimite mail pana la ora 19:00. Interviurile vor fii intre 19:00 si 22:00.
